packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 46.13.0 - September 5, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Fix syntax highlighting issue with String when clauses in Apex switch statements ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #967](https://github.com/forcedotcom/salesforcedx-vscode/issues/967))
```

Comment:
Based on the docs we might want to change this to `... String literals for when values in Apex Switch Statements` (https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_switch.htm)